src/EloquentBuilder.php

```diff
@@ -52,6 +58,16 @@ public function get($columns = ['*'])
         return $builder->getModel()->newCollection($models);
     }
 
+    /**
+     * @param string $columns
+     *
+     * @return int
+     */
+    public function count($columns = '*'): int
+    {
+        return $this->toBase()->getCountForPagination($columns);
```

Comment:
It's more effort, but do you think it's worth changing this round so the logic is in this method and `getCountForPagination()` just calls this?

Reply:
Not sure what you mean by this, do you mean in the `paginate()` function instead of calling `$this->toBase()->getCountForPagination($columns);` call `$this->count();`?

Reply:
That is kind of what I meant but I'm not sure that makes sense - let's leave it as it is.